Changes.md

```diff
@@ -12,6 +12,9 @@ Improvements
 - Viewer : Tool hotkeys now work as toggles for non-exclusive tools. This is currently noticeable on the Camera Tool - pressing <kbd>T</kbd> again will deactivate it.
 - ArnoldShader and ArnoldLight : Added activators and sections to the UI.
 - ImageWriter : Options are only displayed for the file format being written. If the extension is unknown, all format options are shown.
+- Light Editor : Add keyboard shortcuts for selected light.
```

Comment:
Add -> Added. The audience for `Changes.md` are Gaffer users, who aren't typically accustomed to the "imperative mood" we use for commit messages.

Reply:
Fixed and squashed into 82274bb8ffdb5706aa0902f9e92fd6301439bb4f